tests/security/lynis/lynis_setup.pm

```diff
@@ -34,22 +34,24 @@ sub run {
 
     if (is_sle) {
         add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1);
-        zypper_call("in lynis", timeout => 300);
     }
```

Comment:
```patch
-    if (is_sle) {
-        add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1);
-    }
+    add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1) if is_sle;
```

Reply:
You are correct and totally agree (I commented out those line temporarily as I just want to generate the TW baselines then after that I will restore the original code, it is not a good habit)

Reply:
this was really just a tiny style related suggestion, no functional impact

Reply:
> this was really just a tiny style related suggestion, no functional impact

Done, thanks.